dataclients/kubernetes/routegroup.go

```diff
@@ -478,7 +480,34 @@ func splitHosts(hosts []string, domains []string) ([]string, []string) {
 	return internalHosts, externalHosts
 }
 
-func (r *routeGroups) convert(s *clusterState, df defaultFilters, loggingEnabled bool) ([]*eskip.Route, error) {
+// addRouteGroupTLS compares the RouteGroup host list and the RouteGroup TLS host list
+// and adds the TLS secret to the registry if a match is found.
+func (r *routeGroups) addRouteGroupTLS(ctx *routeGroupContext, tls *definitions.RouteTLSSpec) {
+	// Host in the tls section need to explicitly match the host in the RouteGroup
+	hostlist := compareStringList(tls.Hosts, ctx.routeGroup.Spec.UniqueHosts())
+	if len(hostlist) == 0 {
+		ctx.logger.Infof("No matching tls hosts found")
+		return
+	}
+
+	// Skip adding certs to registry since no certs defined
+	if tls.SecretName == "" {
+		ctx.logger.Infof("No tls secret defined for hosts - %s", tls.Hosts)
```

Comment:
So everyone that relies on the default cert will get every time this log line.
Maybe `Infof` -> `Debugf` or something like https://pkg.go.dev/golang.org/x/time/rate#Sometimes as we do in https://github.com/zalando/skipper/blob/master/proxy/proxy.go#L743?

Reply:
Went with the Debug log level here. Let me know if you want to do the alternative.